src/lib.rs

```diff
@@ -18,7 +18,7 @@
 //! ```
 //! use gregorian::{Date, Month::*, Year, YearMonth};
 //!
-//! assert!(Year::new(2020).has_leap_day(), true);
+//! assert!(Year::new(2020).has_leap_day(), "{}", true);
```

Comment:
```suggestion
//! assert!(Year::new(2020).has_leap_day());
```

The `true` actually shouldn't be there :)